pkg/cloudprovider/providers/azure/azure_backoff_test.go

```diff
@@ -0,0 +1,132 @@
+package azure
```

Comment:
missing boilerplate for new files

Reply:
Also should run `hack/update-bazel.sh` for newly added file

Reply:
Both done.